ui/docs/how_to_use_the_ui.md

```diff
@@ -0,0 +1,150 @@
+# Using the ARISE Prediction UI
+
+The ARISE Prediction Tool provides an intuitive web interface for estimating resource requirements and performance metrics for AI workloads. This document explains the features of the ARISE UI, how to configure input and output parameters, interpret results, and use filters effectively. A usage example is also provided to demonstrate its functionality.
+
+---
+
+## Overview of the ARISE Prediction UI
+
+The ARISE Prediction landing page is designed to help users predict resource usage and execution time for AI workloads based on historical data or performance benchmarks. The UI includes:
+
+- Input configuration options.
+- Customizable output parameters.
+- Filtering tools for better result analysis.
+
+---
+
+## Key Sections
+
+- **Prediction Configuration Panel:**  
+  Located in the left sidebar, this panel allows users to define input parameters, output fields, and other configurations required for the prediction process. It also includes the **Predict** button.
+
+- **Input Configuration Panel:**  
+  Enables the selection of input fields. Users can define numeric or categorical fields and set ranges or values to specify the prediction space.
+
+- **Output Configuration Panel:**  
+  Allows the selection of output fields to determine the desired prediction metrics.
+
+- **Main Display Area:**  
+  Displays prediction results and supports additional interactions.
+
+- **Interactive Query Panel:**  
+  Found at the bottom right, this AI Bot lets users ask questions or explore additional features.
+
+---
+
+## Configuring Input Parameters
+
+Input parameters are set in the **Prediction Configuration Panel** and are essential for accurate results. The available fields depend on the dataset and model used. Below are the configurable input parameters for the `MLCommons` dataset:
```

Comment:
@eranra link to the mlcommons example folder in the repo

Reply:
fixed